Aptoide/aptoide-client-v8
# [ASV-1333] Remove old download manager

**What does this PR do?**

This PR aims at removing the old download manager code, which was left in the project so that we could see potential missing requirements.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] InstallService.java

**How should this be manually tested?**

Perform a download and try to interact with the notifications which is where some of the code from the old download manager (constants) was still being used. **Confirm that the download works ok and all analytics events are correctly sent.**

Also, try to do a download on more than one place.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1333](https://aptoide.atlassian.net/browse/ASV-1333)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
